.github/workflows/pocket_bedrock.yaml

```diff
@@ -0,0 +1,43 @@
+name: Pocket on Bedrock
```

Comment:
As mentioned, can we rename at least this to avoid "Bedrock", e.g. "Pocket WWW" or "Pocket Website"? Also, should I add the badge to the README later, or do you plan to add them here?